cluster/images/kube-discovery/README.md

```diff
@@ -0,0 +1,39 @@
+### kube-discovery
+
+An initial implementation of a Kubernetes discovery service using JSON Web Signatures.
+
+This prototype is configured by kubeadm and run within kubernetes itself.
+
+## Requirements
+
+Generate a CA cert save it to: /tmp/secret/ca.pem to run the service or unit tests. (will not be required for unit tests for long) Similarly when run within kubernetes we expect a secret to be provided at this location as well. (see below)
+
+## Build And Run From Source
+
+```
+$ make WHAT=cmd/kube-discovery
```

Comment:
Updated, I swapped in amd64 as that seems overwhelmingly most likely for a person looking to build from source and run it locally.